src/datasets/utils/filelock.py

```diff
@@ -125,8 +132,10 @@ class BaseFileLock:
     Implements the base class of a file lock.
     """
 
-    def __init__(self, lock_file, timeout=-1):
+    def __init__(self, lock_file, timeout=-1, filename_max_length=255):
```

Comment:
Sorry for nitpicking, but maybe `max_filename_length` instead? I leave to your judgment.

Reply:
doing the change right now :)